components/button/button-add.js

```diff
@@ -53,13 +54,26 @@ class ButtonAdd extends PropertyRequiredMixin(FocusMixin(LocalizeCoreElement(Lit
 			}
 
 			.line {
-				border-top: 1px solid var(--d2l-color-mica);
+				background: var(--d2l-color-mica);
```

Comment:
This was switched from border to background in order to do a linear gradient animation (below)